src/medCore/gui/medAbstractWorkspace.cpp

```diff
@@ -105,11 +105,11 @@ medAbstractWorkspace::medAbstractWorkspace(QWidget *parent) : QObject(parent), d
     d->layersToolBox->setTitle("Layer settings");
     d->layersToolBox->hide();
 
-    d->layerListToolBox = new medToolBox(d->layersToolBox);
+    d->layerListToolBox = new medToolBox();
     d->layerListToolBox->header()->hide();
     d->layersToolBox->addWidget(d->layerListToolBox);
 
-    d->interactorToolBox = new medToolBox(d->layersToolBox);
+    d->interactorToolBox = new medToolBox();
```

Comment:
And here.

Remember that d is just a POD struct, there's no cleanup. We might delete the pod, but we don't free any memory tracked by the pointers in the POD.